.github/workflows/bot-cr-gate.yml

```
@@ -0,0 +1,121 @@
+name: Bot — CR Gate
+
+# Fires when a pull_request_review_thread is resolved or unresolved.
+# Manages the unresolved ↔ complete transition based on open CR thread count.
+#
+# On resolved: re-count open CR threads via GraphQL.
+#   If 0 open threads AND label is "coderabbit: unresolved" → swap to complete.
+#
+# On unresolved: if label is "coderabbit: complete" → swap back to unresolved.
+
+on:
+  pull_request_review_thread:
```

actionlint (1.7.12)

[error] unknown Webhook event "pull_request_review_thread". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)